src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -193,6 +214,11 @@ const objectParser = async ({ item, setObject, onPress }) => {
 };
 
 var styles = StyleSheet.create({
+  animationButton: {
+    alignSelf: 'center',
+    bottom: 100,
+    padding: 12
```

Comment:
normalize for some cases? can this conflict somehow with safeareas of particular devices?

Reply:
I'll try it on different devices.